app/views/entities/show.rb

```diff
@@ -83,6 +73,31 @@ def content
       end
     end
 
+    def entity_tab_items
```

Comment:
Similar code is repeated in these hashes, "data.europeana.eu" is repeated in `#entity_query_items_by` and `#entity_query_items_by` only supports one tab.

I recommend replacing `#entity_tab_items`, `#entity_query_items_by` and `#build_proxy_dc` with:
```ruby
    ENTITY_SEARCH_QUERY_FIELDS = {
      agent: {
        by: %w(proxy_dc_creator proxy_dc_contributor)
      },
      concept: {
        about: 'what'
      }
    }.freeze

    def entity_tab_items
      ENTITY_SEARCH_QUERY_FIELDS[api_type].keys.map do |relation|
        entity_tab_items_one_tab(api_type, relation)
      end
    end

    def entity_tab_items_one_tab(api_type, relation)
      search_query = entity_search_query(api_type, relation)
      {
        tab_title: t("site.entities.tab_items.items_#{relation}", name: entity_name),
        url: search_path(q: search_query, format: 'json'),
        search_url: search_path(q: search_query)
      }
    end

    def entity_search_query(api_type, relation)
      fields = ENTITY_SEARCH_QUERY_FIELDS[api_type.to_sym][relation.to_sym]
      [fields].flatten.map do |field|
        %(field: "#{http://data.europeana.eu/#{api_path}}")
      end.join(' OR ')
    end
```

Then we will only need to modify `ENTITY_SEARCH_QUERY_FIELDS` when adding tabs.